src/OrbitSshQt/Task.cpp

```diff
@@ -76,7 +76,8 @@ outcome::result<void> Task::run() {
           emit readyReadStdOut();
         }
         return result.error();
-      } else if (!result) {
+      }
+      if (!result) {
```

Comment:
The rest of the chain should also be split then, because of the `break`s, otherwise this creates an inconsistency that worsens readability.

Reply:
And then then last `else if` can be removed altogether.

Reply:
Done. Also simplified the second last if.